recipe/meta.yaml

```diff
@@ -1,38 +1,33 @@
-{% set version = "4.23.0" %}
+{% set version = "4.25.0" %}
 
 package:
   name: jsonschema
```

Comment:
Could extract the name into jinja variable and use standardized url
`{% set name = "jsonschema" %}`
`name: {{ name|lower }}`
`url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`